addons/misra.py

```diff
@@ -105,6 +105,10 @@ def rawlink(rawtoken):
 }
 
 
+# Reference: N1750 6.5.16
+ASSIGNMENT_OPERATORS = {s + '=' for s in ('', '*', '/', '%', '+', '-', '<<', '>>', '&', '^', '|')} | {'='}
```

Comment:
It would contain `'=' ` twice (if it would not be a set) if I interpret the code correctly. One time because of the empty string and then it is added at the end of the line explicitly. But I could be wrong.

Reply:
Thanks.